egs/wsj/s5/steps/data/reverberate_data_dir.py

```diff
@@ -0,0 +1,352 @@
+#!/usr/bin/env python
+# Copyright 2016  Tom Ko
+# Apache 2.0
+# script to generate reverberated data
+
+# we're using python 3.x style print but want it to work in python 2.x,
+from __future__ import print_function
+import argparse, glob, math, os, random, sys, warnings, copy, imp, ast
+
+train_lib = imp.load_source('ntl', 'steps/nnet3/nnet3_train_lib.py')
```

Comment:
Including the nnet3 library just for `RunKaldiCommand` is not desirable. For now just write `steps/data/data_lib.py` where the `RunKaldiCommand` is replicated. We can think of a better way to share `RunKaldiCommand` later.